docs/install/_rpi.rst

```diff
@@ -1,52 +1,73 @@
 .. _rpi:
 
-Raspbian
-========
+Raspberry Pi
+============
 
-Compatibility
--------------
+There are several varieties of operating systems for Raspberry Pi. This guide is intended for and tested on `Raspian <https://www.raspberrypi.org/>`__, the most popular choice of OS, based on Debian. To obtain and install Raspbian, refer to the official documentation.
+
+Kolibri is intended for **Raspberry Pi Model 3** and upwards.
+
+.. note:: The standard Raspbian choice has a graphical desktop. But you can also install **Raspbian Lite** that uses less resources but only has a command line interface. This documentation works seemlessly on both, but you might find that some packages are already installed in the desktop version of Raspbian.
 
-Kolibri works well on Debian-based distributions for `Raspberry Pi <https://www.raspberrypi.org/>`_, such as Raspbian, and has been tested on RPi 3 models.
 
 Install
 -------
 
-1. Running ``add-apt-repository`` as shown in the PPA instructions does not work. Instead, run::
+#. We need to upgrade the ``python3-cffi`` library, which is outdated on Raspbian. Upgrade it like this::
```

Comment:
FYI I've been trying to migrate all uses of the `::` shorthand to more explicit:

```
.. code-block:: bash

      sudo apt install libffi-dev
      sudo pip3 install pip --upgrade
      sudo pip3 install cffi --upgrade
      sudo systemctl start kolibri
```

This enables syntax highlighting and also avoids syntax that originally confused me when learning RST

Reply:
I know how this works, I've intentionally skipped syntax highlighting for bash, because it was unhelpful. But let's be consistent, I'll use this.